spec/models/provider_spec.rb

```diff
@@ -1,5 +1,5 @@
 describe Provider do
-  let(:provider) { described_class.new }
+  let(:provider) { FactoryGirl.create(:provider) }
```

Comment:
I don't think this should have changed for the rest of the tests when only the two new ones require a record be persisted to the database.  The remaining tests will be slowed down unnecessarily.

Reply:
got them separately handled